src/resources/js/components/layout/SideBar.vue

```diff
@@ -35,7 +37,48 @@ export default {
         path: '/presentation/creer',
       },
     ],
+    notAuthRoutes: [
+      {
+        path: '/login',
+        name: 'Connexion',
+        icon: 'mdi-account-circle',
+      },
+      {
+        path: '/creer-compte',
+        name: 'Création de compte',
+        icon: 'mdi-account-plus',
+      },
+      {
+        path: '/reset',
+        name: 'Mot de passe oublié',
+        icon: 'mdi-account-question',
+      },
+      {
+        path: '/reset-password/:token',
+        name: 'Redéfinir mot de passe',
+        icon: 'mdi-account-convert',
+      },
```

Comment:
The user receives a mail with a link to change the password. A link on the sidebar is not useful.

Reply:
you are right, i put it with the rest too fast, removed in 7f56c13ebaec850c359934a3739022c7050ba86d